src/Filesystem/FilesystemManager.php

```diff
@@ -22,4 +22,16 @@ public function identify($disk)
         }
         return $configName;
     }
+
+    protected function resolve($name, $config = null)
+    {
+        if (is_null($config)) {
+            $config = $this->getConfig($name);
+        }
+        if ($name === 'local') {
```

Comment:
Shouldn't this be the following instead?
```php
if ($config['driver'] === 'local' && empty($config['visibility']))
```

Reply:
That works too